GVFS/GVFS.FunctionalTests/Program.cs

```diff
@@ -108,7 +107,6 @@ public static void Main(string[] args)
 
             RunBeforeAnyTests();

```

Comment:
@kewillford could the code in `RunBeforeAnyTests` be moved to `GlobalSetup.RunBeforeAnyTests`, or would that be called too early?

Reply:
(I think the current changes are good, I was just wondering if this code could be moved too)

Reply:
I will test that and see.  I don't see anything in the code that should prevent it and from my understanding it will be ran at the beginning of the `RunTests` call below but I haven't looked into the code to see what it actually is doing.